x/accesscontrol/keeper/keeper.go

```diff
@@ -227,7 +227,7 @@ func (k Keeper) GetMessageDependencies(ctx sdk.Context, msg sdk.Msg) []acltypes.
 	// Default behavior is to get the static dependency mapping for the message
 	messageKey := types.GenerateMessageKey(msg)
 	dependencyMapping := k.GetResourceDependencyMapping(ctx, messageKey)
-	if dependencyGenerator, ok := k.MessageDependencyGeneratorMapper[types.GenerateMessageKey(msg)]; dependencyMapping.DynamicEnabled && ok {
+	if dependencyGenerator, ok := k.MessageDependencyGeneratorMapper[types.GenerateMessageKey(msg)]; ok {
```

Comment:
why do we want to remove `dynamic enabled`? or just for testing?

Reply:
oh ye, this is just for testing - instead of enabling it everytime through gov proposals